build.rs

```diff
@@ -193,6 +193,9 @@ fn build_mbedtls(nng: &mut cmake::Config, is_ninja: bool) {
     config.define("ENABLE_TESTING", "OFF");
 
     let mut dest = config.build();
+    // On some 64bit Linux distributions the library directory is `lib64`
+    // instead of `lib`. Rename it in the name of consistency.
+    let _ = std::fs::rename(dest.as_path().join("lib64"), dest.as_path().join("lib"));
```

Comment:
While it is ok in general, I believe lib64 is specific to GNU/Linux systems so can you please add `cfg(target_os = "linux")` in both places?

Reply:
Sure thing